### PR TITLE
Native Messaging - Support multiple extensions concurrently

### PR DESCRIPTION
**Pull request by Hinton**
The old implementation assumed that only a single browser extension was running, which resulted in the `sharedSecret` being constantly invalidated. This change assumes the browser sends a unique `appId` which is used to decide which shared secret to use.

Also display an error message informing the user to turn on desktop biometrics before browser biometrics can be enabled.
Windows biometric prompt now grabs focus to ensure it appears above browser windows. 

Depends on https://github.com/bitwarden/jslib/pull/234

### CODE REVIEW

File: src/locales/en/messages.json [hunk 1]
```diff
@@ -1453,9 +1453,15 @@
   "verifyBrowserTitle": {
     "message": "Verify browser connection"
   },
-  "verifyBrowserDescription": {
+  "verifyBrowserDesc": {
```
Comment (cscharf) [inline src/locales/en/messages.json:1456]: Why the change to an abbreviation? Does it need to change?

Comment (Hinton) [reply src/locales/en/messages.json:1456]: We use the abbreviation for all the other dialogs, so seemed best to be consistent.

**Review by cscharf — COMMENTED** [review]
Looks good so far, waiting for final changes.

File: src/main/nativeMessaging.main.ts [hunk 2]
```diff
@@ -19,15 +20,16 @@ export class NativeMessagingMain {
 
         ipc.serve(() => {
             ipc.server.on('message', (data: any, socket: any) => {
-                // This is a ugly hack until electron is updated 7.0.0 which supports ipcMain.invoke
```
Comment (cscharf) [inline src/main/nativeMessaging.main.ts:22]: Does it make sense to remove this comment now?

Comment (Hinton) [reply src/main/nativeMessaging.main.ts:22]: Yes, the implementation was changed slightly in order to support sending multiple messages to the browser instead of only responses. Which is needed to let the browser know we want to validate the fingerprint.

**Review by cscharf — DISMISSED** [review]

**Comment by cscharf** [comment]
@Hinton , can you update jslib for this so when merged will successfully build? The jslib change has been approved and merged.

**Comment by Hinton** [comment]
@cscharf Done, thanks.

File: src/locales/en/messages.json [hunk 3]
```diff
@@ -1453,15 +1453,33 @@
...
     "message": "Please ensure the shown fingerprint is identical to the fingerprint showed in the browser extension."
   },
+  "biometricsNotEnabledTitle": {
+    "message": "Biometrics not enabled"
+  },
+  "biometricsNotEnabledDesc": {
+    "message": "Browser biometric requires desktop biometric to be enabled in the settings first."
```
Comment (cscharf) [inline src/locales/en/messages.json:1481]: Should read: `"Browser biometrics requires desktop biometrics to be enabled in the settings first."` (biometrics should be plural)

**Review by cscharf — COMMENTED** [review]
one quick change needed, either missed it from before or it reverted.

**Review by cscharf — APPROVED** [review]
